src/explorer/collectionTreeItem.ts

```diff
@@ -381,6 +383,8 @@ export default class CollectionTreeItem extends vscode.TreeItem
             return resolve(false);
           }
 
+          this.isDropped = true;
```

Comment:
Is `!successfullyDroppedCollection` something we ever have to worry about? I can’t see how it wouldn’t fail with an error but still not work

Reply:
Oh, good point! Yeah, maybe it's not really possible in this case to get a false there, but better safe than sorry

Reply:
Done in cd289f3